config-default.yml

```
@@ -25,7 +25,7 @@ style:
         green_chevron: "<:greenchevron:418104310329769993>"
         red_chevron:   "<:redchevron:418112778184818698>"
         white_chevron: "<:whitechevron:418110396973711363>"
-        lemoneye2:     "<:lemoneye2:435193765582340098>"
+        bb_message:     "<:bbmessage:472476937504423936>"
```

I think this will fix misalignment? Hard to tell on GH.

```suggestion
        bb_message:    "<:bbmessage:472476937504423936>"
```